packages/shared/src/components/modals/SquadTourModal.tsx

```diff
@@ -1,16 +1,17 @@
 import React, { ReactElement } from 'react';
 import CloseButton from '../CloseButton';
-import SquadTour, { CarouselMinimalProps } from '../squads/SquadTour';
+import SquadTour from '../squads/SquadTour';
 import { Modal, ModalProps } from './common/Modal';
 import { ButtonSize } from '../buttons/Button';
+import { useSquadTour } from '../../hooks/useSquadTour';
 
 function SquadTourModal({
   onRequestClose,
-  onScreenIndexChange,
   ...props
-}: ModalProps & CarouselMinimalProps): ReactElement {
+}: ModalProps): ReactElement {
+  const { onTourIndexChange } = useSquadTour();
   const onModalClose: typeof onRequestClose = (param) => {
-    onScreenIndexChange?.(-1);
+    onTourIndexChange(-1);
     onRequestClose(param);
   };
```

Comment:
@sshanzel Note I made a new hook for this in favor of individual implementations 👯

Reply:
Whoa. That would make it much cleaner! I will wait for that PR to get merged before proceeding here.